### DISCUSSION TITLE
Lime test windows is giving errors on Kade Engine 1.7

### DISCUSSION THREAD

**Discussion by ItsJayden2270** [KadeArchive/Kade-Engine#2383] [Q&A]
I have no idea how this works but when I downloaded everything for haxelpixel and stuff I tried building the game this happens 
![image](https://user-images.githubusercontent.com/91494476/134964675-87d29845-7b55-4f70-a7ed-4deb86e577f9.png)

**Comment by JOELwindows7** [comment]
# There is problem with `linc_luajit` https://github.com/nebulazorua/linc_luajit
https://github.com/KadeDev/Kade-Engine/issues/2375 bug issue

it's not your fault. I have the same problem too. Please wait for fix
